src/createServer.js

```diff
@@ -1,3 +1,29 @@
-// Write code here
-// Also, you can create additional files in the src folder
-// and import (require) them here
+const http = require('http');
+const { parseRequestUrl, validateData } = require('./utils');
+const { sendErrorResponse, sendSuccessResponse } = require('./reponseHandlers');
+
+function createServer() {
+  const server = http.createServer((request, response) => {
+    if (request.url !== '/favicon.ico') {
```

Comment:
How about reverting this condition? The rest of the function wouldn't have to be nested
```suggestion
    if (request.url === '/favicon.ico') return;
```